<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{

}

QLabel{
color: #E5E5E5;
border: #FFE5B4;
border-radius: 1px;
font-family: 'Roboto', sans-serif;
font-size: 16px; /* Tamanho do texto */
line-height: 1.5; /* Altura da linha para melhor legibilidade */
}
QLabel#label_title{
color: #E5E5E5;
border: #FFE5B4;
border-radius: 1px;
font-family: 'Roboto', sans-serif;
font-size: 30px; /* Tamanho do texto */
line-height: 1.5; /* Altura da linha para melhor legibilidade */
}

QLineEdit{
background:Silver;
border: 2px rgb(85, 170, 255) solid;
border-radius: 10px;
color: #B87333
}
QLineEdit:focus {
Background: #F4FDFF;
border: 2px  solid #FFE5B4;
color: #B87333
}

QComboBox{
background:Silver;
border: 2px rgb(85, 170, 255) solid;
border-radius: 10px;
color: #B87333
}

QPushButton{
background:#808B8E;
border: 2px rgb(85, 170, 255) solid;
border-radius: 10px;
color: #E5E5E5
}

QPushButton:hover{
background:#43d7f7;
border: 2px rgb(85, 170, 255) solid;
border-radius: 10px;
color: black
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_title">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Roboto,sans-serif</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="text">
     <string>TechBooks</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>290</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Usuário ou Email</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_usuario">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>320</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>360</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Senha</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_login">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>430</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Entrar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_senha">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>390</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_cadastro">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>560</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Cadastre-se</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>230</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Identificação Conta</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>70</y>
      <width>151</width>
      <height>151</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>book-reader.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
